docs/appendix-A.rst

```diff
@@ -0,0 +1,1783 @@
+.. Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+   other Shroud Project Developers.
+   See the top-level COPYRIGHT file for details.
+
+   SPDX-License-Identifier: (BSD-3-Clause)
+
+.. All of the examples are ordered as
+   C or C++ code from users library
+   YAML input
+   C++ wrapper
+   Fortran interface
+   Fortran wrapper
+   Fortran example usage
+   C++ example usage
+
+Sample Fortran Wrappers
+=======================
+
+This chapter gives details of the generated code.
+It's intended for users who want to understand the details
+of how the wrappers are created.
+
+All of these examples are derived from tests in the ``regression``
+directory.
+
+.. _example_NoReturnNoArguments:
+
+No Arguments
+------------
+
+C library function:
+
+.. literalinclude:: ../regression/run/clibrary/clibrary.c
+   :language: c
+   :start-after: start NoReturnNoArguments
+   :end-before: end NoReturnNoArguments
+
+YAML:
+
+.. code-block:: yaml
+
+    - decl: void NoReturnNoArguments()
+
+Calls C via the interface:
```

Comment:
It might be helpful here to say that this is Fortran code.  E.g. 
> Fortran calls C via the following interface...`

Reply:
I changed all of the similar sections to use your suggestion.